test/test_bin/test_check_mta_sts.py

```diff
@@ -0,0 +1,54 @@
+import unittest
+import boto3
+from botocore.exceptions import 
+from unittest.mock import patch
+
+import check_mta_sts
+
+class TestCheckMtaSts(unittest.TestCase):
+    
+    @patch('check_mta_sts.boto3.client')
+    def test_s3_initialization(self, mock_client):
```

Comment:
The code here doesn't execute any of the code under test 🧪

Ideally the code, at some point, should call a method within the imported module i.e. `check_mta_sts.main()` and verify assert against the return value of the function, or the side affects of the function 😁

A great methodology that stuck with me is the [Given, When, Then](https://martinfowler.com/bliki/GivenWhenThen.html) structure for writing tests 🏗️ 

so the tests will loosely look like the following:

```python
// Given
// ...set up the environment to test like mock repsonses

// When
// ... call the method to test i.e. response = check_mta_sts.main()

// Then
// ... assert some values on the response or side effects i.e. self.assertEquals(response, 1)
```